R/rstudio.R

```diff
@@ -10,10 +10,13 @@
 #'   * Adds RStudio files to `.gitignore`
 #'   * Adds RStudio files to `.Rbuildignore`, if project is a package
 #'
+#' @param os Operating system; used to set default line endings.
 #' @export
-use_rstudio <- function() {
+use_rstudio <- function(os = .Platform$OS.type) {
```

Comment:
Is there a reason you use `.Platform$OS.type` instead of Sys.info()$sysname`? In practice I guess it doesn't much matter, they should always be equivalent. Though I guess `.Platform` is set when R is compiled and `Sys.info()` is looked up at runtime.

Reply:
Because I didn't know about `Sys.info()`? 😄  (But I do think `OS.type` is slightly more appropriate here)